crates/swc_ecma_minifier/src/option/terser.rs

```diff
@@ -69,6 +71,22 @@ pub enum TerserTopRetainOption {
     Seq(Vec<Atom>),
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+#[non_exhaustive]
+pub struct TerserExperimentalOptions {
+    #[serde(default = "true_by_default")]
+    pub reduce_escaped_newline: bool,
```

Comment:
```suggestion
    pub reduce_escaped_newline: Option<bool>,
```
Sorry, I missed this one in the previous review.

Reply:
`true_by_default` with `bool` means

 - `experimental: {}` => `true`, regardless of `defaults`
 - `experimental: null` => Uses `defaults`

Reply:
I think those two should behave identically because the user didn't specify the  `reduce_escaped_newline`

Reply:
It's my oversight. Thanks to point out.